apps/src/codebridge/Workspace/index.tsx

```diff
@@ -44,10 +45,18 @@ const Workspace = () => {
             : 'You are editing start sources.'}
         </div>
       )}
-      <Editor
-        langMapping={config.languageMapping}
-        editableFileTypes={config.editableFileTypes}
-      />
+      <div
+        className={moduleStyles.workspaceWorkarea}
+        style={{
+          gridTemplateColumns: config.showFileBrowser ? '200px auto' : '100%',
+        }}
+      >
+        {config.showFileBrowser && <FileBrowser />}
+        <Editor
+          langMapping={config.languageMapping}
+          editableFileTypes={config.editableFileTypes}
+        />
+      </div>
```

Comment:
The new div here wraps up the `FileBrowser` and `Editor` into a 200px/100% grid, depending upon the `showFileBrowser` boolean.

Maybe there's a slicker way to lay this out with a flexbox? Maybe specifying a width on the FileBrowser? I don't think our `PanelContainer`s take widths, so that'd probably involve more refactoring.